### ISSUE TITLE
nvrtc failed: CUDA_ERROR_INVALID_IMAGE

### ISSUE THREAD

**Issue by canselcik** [nanopool/nanominer#9]
```
_ __   __ _ _ __   ___  _ __ ___ (_)_ __   ___ _ __ 
| '_ \ / _` | '_ \ / _ \| '_ ` _ \| | '_ \ / _ \ '__|
| | | | (_| | | | | (_) | | | | | | | | | |  __/ |   
|_| |_|\__,_|_| |_|\___/|_| |_| |_|_|_| |_|\___|_|   

Version 1.1.0
2019-03-10 15:28:26: CUDA driver version is 10.0, runtime version is 9.0
Single device detected
GPU 0 PCI 01:00.0, Platform: CUDA, Name: GeForce RTX 2080 Ti, 10989 MB available
2019-03-10 15:28:26: Started EthMan server on port 3333 without a password (read-only mode).
2019-03-10 15:28:26: Started web interface on port 9090 without a password.
----------------CryptoNightR:---------------
Currency: Monero
Wallet: 491YjJCyTuqK...s5cXVtkAvdeCemn5e7v8K5
Rig name: rig1
Email was not set.
Hashrate threshold to restart miner is not set
Added pool: pool.minexmr.com:4444
Auto pool sorting is disabled
Using all GPU devices
-------------Common parameters:-------------
Log path: ./logs/
Log is written to file
GPU watchdog is enabled
Never calling reboot.sh (always restarting the miner in case of errors). 
2019-03-10 15:28:27: Setting up algorithm: CryptoNightR
2019-03-10 15:28:28: Connecting to pools...
2019-03-10 15:28:29: Connected to pool: pool.minexmr.com:4444
2019-03-10 15:28:30: New job from pool.minexmr.com:4444 | share difficulty = 15000
Monero - Total speed: 0.000 H/s, Total shares: 0 Rejected: 0, Time: 00:02
GPU0 t=67°C fan 71% power 94.67W. Total power: 94.67W
2019-03-10 15:28:30: nvrtc failed
2019-03-10 15:28:30: System exception: 
2019-03-10 15:28:30: device: 0; CUDA_ERROR_INVALID_IMAGE
```

Having this issue on `Linux compact 4.20.6-arch1-1-ARCH #1 SMP PREEMPT Thu Jan 31 08:22:01 UTC 2019 x86_64 GNU/Linux`.

Searching online, I found that this may happen if the program is built for 32-bit but running on 64-bit however that doesn't seem to be the case `nanominer: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, Go BuildID=_J_UuhwfEiufxQuQhv0F/Pr1vaafOTb4fF25GUYJB/5vUdnBUAn0ufn-jlOX2_/p55tuiY-b-YOwABn6633, BuildID[sha1]=218321560f595545da9d8bca11d07201beb3fecb, stripped`.

**Comment by Grumpy-Dwarf** [comment]
Thanks for reporting. That's because we use CUDA 9 in a way which is incompatible with new GeForce 20 series. Will switch to CUDA 10 soon to support these GPUs.

**Comment by Grumpy-Dwarf** [comment]
FIxed in [1.2.0 release](https://github.com/nanopool/nanominer/releases/tag/v1.2.0)
